_people/StephenGuy.md

```diff
@@ -0,0 +1,12 @@
+---
+first_name: Stephen J.
+last_name: Guy
+title: Associate Professor of Computer Science and Engineering
+school: University of Minnesota
+website: https://www-users.cs.umn.edu/~sjguy/
+image: https://www-users.cs.umn.edu/~sjguy/img/Stephen2012.jpg
+---
+Prof. Guy directs the [Applied Motion Lab](http://motion.cs.umn.edu). His research focuses on the development of artificial intelligence for use in autonomous robotics (e.g., collision avoidance and path planning under uncertainty) and computer simulations of human movement and behavior (e.g., crowd simulation and virtual characters).
+- Guy, Stephen J., et al. "[Clearpath: highly parallel collision avoidance for multi-agent simulation.](http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.148.7460&rep=rep1&type=pdf)" Proceedings of the 2009 ACM SIGGRAPH/Eurographics Symposium on Computer Animation. 2009.
+- Karamouzas, Ioannis, Brian Skinner, and Stephen J. Guy. "[Universal power law governing pedestrian interactions.](https://arxiv.org/pdf/1412.1082.pdf)" Physical review letters. 2014.
```

Comment:
Link to physical review letters not arxiv

Reply:
See revisions.